doc/tutorials/pixel_data/compressing.rst

```diff
@@ -1,230 +1,425 @@
-========================
-Compressing *Pixel Data*
-========================
+====================================================
+*Pixel Data* - Part 3: Compression and decompression
+====================================================
 
 .. currentmodule:: pydicom
 
-This tutorial is about compressing a dataset's *Pixel Data* and covers
-
-* An introduction to compression
-* Using data compressed by third-party packages
-* Compressing data using *pydicom*
-
-It's assumed that you're already familiar with the :doc:`dataset basics
-<../dataset_basics>`.
 
+In part 1 of this tutorial you learnt how to :doc:`access the pixel data
```

Comment:
"learned" in American English, same below and in the other parts

Reply:
Well, I've learned something today.